Makefile

```diff
@@ -127,6 +127,52 @@ restart-celery:
 		sudo systemctl status --no-pager celeryworker.service) & \
 	fi
 
+check-local-hub:
+	@echo "Checking LOCAL_HUB setting..."
+	@if [ "$$(uname)" = "Darwin" ]; then \
+		grep -E '^LOCAL_HUB=' .env | cut -d'=' -f2 | tr -d '"' | tr -d ' ' > .hub_setting; \
+	else \
+		grep -oP 'LOCAL_HUB=\K.*' .env | tr -d '"' | tr -d ' ' > .hub_setting; \
+	fi
+	@echo "LOCAL_HUB is $$(cat .hub_setting)"
+
+restart-hub: check-local-hub
```

Comment:
Unless `check-local-hub` has a specific reason to exist as a separate target, I'd suggest to move its code in here. That way we avoid leaving behind possible leftovers like `.hub_setting` file

Reply:
other that that, LGTM